src/app/(contents)/post/[detailId]/PostDetail.tsx

```
@@ -0,0 +1,203 @@
+                toastCall("포스팅을 삭제하지 못했습니다.", "error")
+            })
+    }
+
+    const diff = dayjs().diff(dayjs(postDetail.createdAt), "hour")
+
+    useEffect(() => {
+        getPostDetail(Number(id))
+    }, [id, doEdit])
+
+    // HTML 파싱 및 img 태그 변환 함수
+    const parseHtmlWithNextImage = (htmlContent: string) => {
```

ESLint

[error] Missing return type on function.

(@typescript-eslint/explicit-function-return-type)